Sources/PostgresNIO/New/PostgresCell.swift

```diff
@@ -0,0 +1,47 @@
+import NIOCore
+
+struct PostgresCell: Equatable {
+    var bytes: ByteBuffer?
+    var dataType: PostgresDataType
+    var format: PostgresFormat
+
+    var columnName: String
+    var columnIndex: Int
+
+    /// use this only for testing
```

Comment:
There are no other initializers and the type is not `Codable` - this is the only way to create one of these. I'm wondering if this is potentially an example of splitting up the PRs _too_ much- if there're missing pieces that you couldn't even add via extension (such as non-delegating initializers), the PR is not complete 😕.

Or the comment may just be outdated, but then what would I make a fuss about? 😉

Reply:
The comment was outdated. Removed. 🤦‍♂️